config/locales/client.en.yml

```diff
@@ -87,12 +87,12 @@ en:
 
       create_channel:
         choose_category:
-          label: "Choose a category to manage who can access the channel"
+          label: 'Choose a category<br>Manage access by visiting <a href=%{link} target="_blank">%{category} security settings</a>'
```

Comment:
I can't remember but are category names escaped by default? Otherwise, this may result in an XSS.